dplutils/observer/mlflow.py

```diff
@@ -25,13 +25,15 @@ class MlflowObserver(Observer):
             ``mlflow.MlflowClient.create_run``.
     """
     def __init__(self, run=None, experiment=None, tracking_uri=None, **mlflow_kwargs):
+        if mlflow is None:
+            raise ImportError("mlflow must be installed to create observer run!")
+
```

Comment:
Good spot, I missed that this should have been hoisted. We sure it shouldn't also be hoisted for aim also?

Reply:
I think it would be unnecessarily restrictive, since for aim it is targeted to the only place package is actually needed. Technically someone could use a compatible object that does not come from aim

Reply:
>Technically someone could use a compatible object that does not come from aim

👍